airflow/jobs/job.py

```diff
@@ -201,12 +215,15 @@ def heartbeat(
 
             job = Job._update_heartbeat(job=self, session=session)
             self._merge_from(job)
-
+            time_since_last_heartbeat = (timezone.utcnow() - previous_heartbeat).total_seconds()
+            if time_since_last_heartbeat > self.health_check_threshold:
+                self.log.info("INFO: Heartbeat recovered after %.2f seconds", time_since_last_heartbeat)
```

Comment:
We do not need to add the "INFO:". It will be taken care by the logger

Reply:
i will fix this part @amoghrajesh